### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by palica**
please update Funtoo to 1.4 release, this should also fix the problem with metadata.xml showing incorrect information about funtoo vs gentoo.

https://github.com/repology/repology/issues/925

### CODE REVIEW

**Comment by AMDmi3** [comment]
I won't accept dropping metadata. Homepage information is crucial for spitting conflicting projects.

**Comment by danielrobbins** [comment]
@AMDmi3 homepage information should be coming from the md5-cache or the ebuild, it is not stored in the metadata.It is fine to drop the metadata, I checked your parser source code.

**Comment by AMDmi3** [comment]
Homepage also includes information from `<upstream>`.

**Comment by danielrobbins** [comment]
@AMDmi3 OK, well I can do a patch so that there is a way through the YAML to disable parsing of just the maintainer info from the metadata.xml. Would that be accepted by you? If so, can you provide info on how you would like it to work and I will submit the patch.

**Comment by AMDmi3** [comment]
The point is that you don't need do hide Funtoo maintainers at all for Repology, see #925 amd #655.

**Comment by danielrobbins** [comment]
OK, gotcha. We can re-send the patch with the metadata.xml still in it as it contains updates for Funtoo 1.4.

I suspect this issue is what is described in #655.

File: repos.d/gentoo/funtoo.yaml [hunk 1]
```diff
@@ -2,45 +2,51 @@
...
-        - ml-lang-kit
-        - net-kit
-        - nokit
+        - dev-kit
         - python-modules-kit
-        - ruby-kit
-        - science-kit
-        - security-kit
-        - text-kit
+        - nokit
```
Comment (AMDmi3) [inline repos.d/gentoo/funtoo.yaml]: Please keep this sorted to reduce diff

File: repos.d/gentoo/funtoo.yaml [hunk 2]
```diff
@@ -2,45 +2,51 @@
...
+        - net-kit
+        - text-kit
+        - science-kit
         - games-kit
-        - haskell-kit
+        - perl-kit
         - java-kit
-        - kde-kit
-        - lang-kit
+        - ruby-kit
```
Comment (AMDmi3) [inline repos.d/gentoo/funtoo.yaml]: You must not duplicate repositories which we handle separately (due to different branch names) below here.

**Comment by AMDmi3** [comment]
Also, does it fully obsolete 1.3? If there's a change that some people still use 1.3, we may leave it there for informational purpose.

**Comment by palica** [comment]
according to this you should be able to squash when merging. https://github.blog/2016-04-01-squash-your-commits/

**Comment by palica** [comment]
this is the documentation for it https://help.github.com/en/articles/about-pull-request-merges

**Comment by AMDmi3** [comment]
> you should be able to squash when merging

I'd prefer to merge properly prepared commit as is. Squashing via GH requires me to edit the commit message and it may end up not what a contributor expects. Also I suspect it would overwrite committer field in the commit.

File: repos.d/gentoo/funtoo.yaml (hunk 1 above)
Comment (palica) [reply repos.d/gentoo/funtoo.yaml]: should be fixed now

File: repos.d/gentoo/funtoo.yaml (hunk 2 above)
Comment (palica) [reply repos.d/gentoo/funtoo.yaml]: should be fixed now

**Comment by palica** [comment]
rebase done

**Comment by AMDmi3** [comment]
Ok, looks good. So what about leaving 1.3 be for now? @danielrobbins?

**Comment by danielrobbins** [comment]
@AMDmi3 if you have to choose one, choose 1.4, since that is the current release, but if you can include both then it is fine to include 1.3 since it is still supported. Thanks.

**Pull request merged**